insertPokemon.php

```diff
@@ -2,15 +2,6 @@
 
 require('functions.php');
 
-$dataBase = getDatabaseObject();
-
-$pokedexNumber = $_POST['addPokedexNo'];
-$pokemonName = $_POST['addPokemonName'];
-$pokemonType = $_POST['addPokemonType'];
-$pokemonImage = $_POST['addPokemonImage'];
-
-$insert = $dataBase->prepare('INSERT INTO `kanto_pokedex` (`pokedex_no`, `name`, `type`, `img_source`) VALUES (?, ?, ?, ?);');
-
-$insert->execute([$pokedexNumber, $pokemonName, $pokemonType, $pokemonImage]);
+addPokemonToDatabase();
```

Comment:
I suggest that you check the return type of addPokemonToDatabase() and only forward the user onto the page pokedexDataBaseProject.php if its true ie if you successfully inserted something into the DB

Reply:
no need as the input requires set values to be inputted and something can only be submitted once those values have all been completed

Reply:
Mike suggested its best practise, and that as a user i might expect to see a 'Success' message after submitting a new entry or a Failure/Try Again message if the DB didnt like the submitted values